src/System.Security.Cryptography.Xml/tests/XmlLicenseEncryptedRef.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class XmlLicenseEncryptedRef : IRelDecryptor
+    {
+        List<AsymmetricAlgorithm> asymmetricKeys = new List<AsymmetricAlgorithm>();
+
+        public XmlLicenseEncryptedRef()
+        {
+        }
+
+        public void AddAsymmetricKey(AsymmetricAlgorithm key)
+        {
+            if (key == null)
+                throw new ArgumentNullException(nameof(key));
+
+            asymmetricKeys.Add(key);
+        }
+
+        private static bool ArrayEqual(byte[] a, byte[] b)
```

Comment:
You already have `using System.Linq;`; why not just use the SequenceEqual extension method, given that this is an early aborter?